app/src/main/java/edu/grinnell/appdev/grinnelldirectory/APICaller.java

```diff
@@ -0,0 +1,138 @@
+package edu.grinnell.appdev.grinnelldirectory;
+
+import android.util.Log;
+
+import java.io.IOException;
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.APICallerInterface;
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.DatabaseAPI;
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+import edu.grinnell.appdev.grinnelldirectory.Model.User;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+
+
```

Comment:
Please use Javadoc documentation to give other developers an idea of what this class does.